### ISSUE TITLE
Warum wurde das Popover nicht mit nativen Mitteln umgesetzt?!

### ISSUE THREAD

**Issue by deleonio** [public-ui/kolibri#8602]
Es muss auch folgendes Verhalten sichergestellt werden: https://github.com/public-ui/kolibri/issues/8443

**Comment by BF150** [comment]
1. es wird das native popover genutzt 
<img width="464" height="163" alt="Image" src="https://github.com/user-attachments/assets/2662d61e-646d-4cac-98cf-0f7e1d48e4f2" />

2. es werden die nativen Funktionen verwendet
3. Allerdings wird vor show/hide das popover versteckt damit die Berechnung der Position nicht zu Flackern führt. Das könnte man mit Anchors dann in Zukunft eleganter lösen
